comp/etw/impl/etwSession.go

```diff
@@ -44,6 +44,13 @@ func (e *etwSession) EnableProvider(providerGUID windows.GUID) error {
 	}
 
 	cfg := e.providers[providerGUID]
+	var pids *C.ULONGLONG
+	var pidCount C.ULONG
+	if len(cfg.PIDs) > 0 {
+		pids = (*C.ULONGLONG)(unsafe.Pointer(&cfg.PIDs[0]))
```

Comment:
nit: prefer using `unsafe.SliceData` instead of `&cfg.PIDS[0]`
https://pkg.go.dev/unsafe#SliceData

Reply:
Interesting. SliceData had been introduced in 1.20 only. Which we already use. Still strange that the previous expression "screw" it up because pidCount was zero anyway.